src/mpyl/cli/build.py

```diff
@@ -270,6 +280,38 @@ def by_choice(choice: Choice):
         return release.tag_name
 
 
+def get_test_releases():
+    console = Console()
+    console.log("Fetching MPyL releases..")
+    url = "https://test.pypi.org/pypi/mpyl/json"
+    data = requests.get(url, timeout=30).json()
+    versions = list(data["releases"].keys())
+    versions.sort(key=packaging_version.Version, reverse=True)
+    versions = [version[:-4] for version in versions[:100]]
+    versions = list(dict.fromkeys(versions))
+    versions = [version + "*" for version in versions]
+    return versions
```

Comment:
Can you show us the structure of `data`, maybe there is a simpler way to get to the same result

Reply:
This looks more like `versions`?

Reply:
Yes, that's what you want right?